tests/api/imagesAPI.test.js

```diff
@@ -0,0 +1,16 @@
+const Images  = require("../../models/ImagesAPI")
+
+describe('Test CAT API endpoints', () => {
+    const images = new Images();
+
+    test('Search cat images', async() => {
+        const response = await images.searchImages();
+        expect(response.status).toEqual(200);       
+    })
+    test('Get a cat image', async() => {
```

Comment:
Update the test naming to reflect what should be the test all about. Example: Verify cat search image endpoint return 200